.travis.yml

```diff
@@ -5,26 +5,15 @@ python:
     - 3.5
     - 3.6
```

Comment:
I believe that it worth to add `3.7` as well.

Reply:
I agree, but I will do that in a follow-up PR for easier tracking 👍

Reply:
#44.